resources/smcp-templates/v1.1/base

```diff
@@ -3,7 +3,7 @@ kind: ServiceMeshControlPlane
 spec:
   istio:
     global:
-      tag: 1.1.11
+      tag: 1.1.12
```

Comment:
Do you also need to update v2.0 files?

Reply:
Yes, good catch

Reply:
I've just checked and it doesn't seem to be included in the 2.0 base

Reply:
sorry, it is but a different name.  I'll update now

Reply:
@jwendell can you check again?  I've just pushed up the 2.0 changes matching the above